### PR TITLE
Sync with develop

### PR DESCRIPTION
**Pull request by Danielxs01**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -12,7 +12,7 @@ buildscript {
 apply plugin: 'cam72cam.universalmodcore'
 apply plugin: 'maven'
 
-String version = '1.0.2'
+String version = '1.0.3'
```
Comment (MarkenJaden) [inline build.gradle:15]: There is no need to change the version here anymore. It's handled by the release pipeline

File: src/main/java/net/landofrails/landofsignals/LandOfSignals.java [hunk 2]
```diff
@@ -28,7 +28,7 @@ public class LandOfSignals extends ModCore.Mod {
     @SuppressWarnings({"java:S1845"})
     public static final String MODID = "landofsignals";
     // Current version
-    public static final String VERSION = "1.0.2";
+    public static final String VERSION = "1.0.3";
```
Comment (MarkenJaden) [inline src/main/java/net/landofrails/landofsignals/LandOfSignals.java:31]: There is no need to change the version here anymore. It's handled by the release pipeline

File: src/main/java/net/landofrails/landofsignals/Mod.java [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 import cam72cam.mod.ModCore;
 
-@net.minecraftforge.fml.common.Mod(modid = Mod.MODID, name = "LandOfSignals", version = "1.0.2", dependencies = "required-after:universalmodcore@[1.1,1.2)", acceptedMinecraftVersions = "[1.12,1.13)")
+@net.minecraftforge.fml.common.Mod(modid = Mod.MODID, name = "LandOfSignals", version = "1.0.3", dependencies = "required-after:universalmodcore@[1.1,1.2)", acceptedMinecraftVersions = "[1.12,1.13)")
```
Comment (MarkenJaden) [inline src/main/java/net/landofrails/landofsignals/Mod.java:5]: Mod.java is auto-generated by UMC, no need to change it here either

File: src/main/resources/mcmod.info [hunk 4]
```diff
@@ -3,7 +3,7 @@
     "modid": "landofsignals",
     "name": "LandOfSignals",
     "description": "Adds fancy railroad signals, signs, decor and other stuff to the game. Use best in combination with a train mod.",
-    "version": "1.0.2",
+    "version": "1.0.3",
```
Comment (MarkenJaden) [inline src/main/resources/mcmod.info:6]: I'll add this to the release pipeline too. So we'll just need to change it in VERSION

**Review by MarkenJaden — COMMENTED** [review]
No need to change the version everywhere, release pipeline handles everything

**Comment by Danielxs01** [comment]
I'll do it anyways. 👍🏼
jk.. fine, I'll do it the other way next time.

**Pull request merged**